etc/bash_completion.d/brew

```diff
@@ -52,8 +53,8 @@ __brew_complete_formulae() {
 
 __brew_complete_installed() {
   local cur="${COMP_WORDS[COMP_CWORD]}"
-  local inst="$(ls "$(brew --cellar)")"
-  COMPREPLY=($(compgen -W "$inst" -- "$cur"))
+  local inst=("$(brew --cellar)"/*)
+  COMPREPLY=($(compgen -W "${inst[*]#$(brew --cellar)/}" -- "$cur"))
```

Comment:
Ideally yes, but unfortunately `compgen -W` only takes a single argument, which it then splits by the value(s) of `IFS`. `inst[@]` expands to multiple values, which causes this command to fail.